packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -27,6 +28,8 @@ const SCHEMA_FILENAME = 'schema.graphql';
 const SCHEMA_DIR_NAME = 'schema';
 const PROVIDER_NAME = 'awscloudformation';
 
+const LAYER_MAPPING_URL = 'https://amplify-rds-layer-resources.s3.amazonaws.com/rds-layer-mapping.json';
```

Comment:
This file will be periodically updated with the latest layer information. Clients will use this to determine which layer to use for SQL lambda. If the file is not accessible, it will fallback to the hardcoded version mapping.

Reply:
How do we update this S3 object? Is it upon every new layer version we publish from our pipeline or every API plugin release?

Reply:
At this point, it is manual update. Once we add integration tests for the RDS layer, we can confidently automate this.